ibm/resource_ibm_private_dns_permitted_network.go

```diff
@@ -183,7 +179,7 @@ func resourceIBMPrivateDnsPermittedNetworkExists(d *schema.ResourceData, meta in
 	_, response, err := sess.GetPermittedNetwork(getPermittedNetworkOptions)
 	if err != nil {
 		if response != nil && response.StatusCode == 404 {
-			return false, nil
+			return false, fmt.Errorf("Error fetching pdns permitted network:%s\n%s", err, response)
 		}
```

Comment:
Don't return err if response != nil && response.StatusCode == 404 {
}
keep the the old code only
return false, nil